src/execution/index/art/art.cpp

```diff
@@ -15,7 +15,8 @@ namespace duckdb {
 ART::ART(const vector<column_t> &column_ids, TableIOManager &table_io_manager,
          const vector<unique_ptr<Expression>> &unbound_expressions, IndexConstraintType constraint_type,
          DatabaseInstance &db, idx_t block_id, idx_t block_offset)
-    : Index(IndexType::ART, table_io_manager, column_ids, unbound_expressions, constraint_type), db(db) {
+    : Index(IndexType::ART, table_io_manager, column_ids, unbound_expressions, constraint_type), db(db),
+      estimated_art_size(0), estimated_key_size(16) {
```

Comment:
I have a PR almost ready for the exact size, but I am still busy writing the tests for that. This is a perfect intermediate solution for #4967.